app/Http/Controllers/API/TraderController.php

```diff
@@ -114,7 +114,12 @@ public function showVouchers(Trader $trader)
                 ]);
             case 'application/json':
             default:
-                return response()->json($vouchers, 200);
+                return response()->json($vouchers->map(function($voucher){
```

Comment:
Check syntax. You need a few more spaces and indents here.

You could also take the creation of the array out above the return to make it slightly less squished into closures.